File: arreglo.py
class Ventas:
    def __init__(self):
      
        self.ventas = [
            [500, 300, 150],  
            [400, 250, 200],  
            [350, 450, 300],  
            [600, 350, 250],  
            [700, 300, 350],  
            [550, 400, 450],  
            [600, 500, 400],  
            [650, 550, 300],  
            [620, 350, 250],  
            [590, 300, 300],  
            [560, 270, 200],  
            [700, 350, 150]   
        ]
       
        self.meses = [
            "Enero", "Febrero", "Marzo", "Abril", "Mayo", "Junio",
            "Julio", "Agosto", "Septiembre", "Octubre", "Noviembre", "Diciembre"
        ]
        
        self.departamentos = ["Ropa", "Deportes", "Juguetería"]

    def buscar_ventas(self, mes, departamento):
      
        if 0 <= mes < 12 and 0 <= departamento < 3:
            return self.ventas[mes][departamento]
        else:
            print("Mes o departamento fuera de rango.")
            return None

    def eliminar_ventas(self,mes, departamento):
      
      if 0 <= mes < 12 and 0 <= departamento < 3:
            self.ventas[mes][departamento] = 0
      else:
            print("Mes o departamento fuera de rango.")

    def imprimir_ventas(self):
        
        print(f"{'Mes':<12} {'Ropa':<10} {'Deportes':<10} {'Juguetería':<10}")
        for i in range(12):
            print(f"{self.meses[i]:<12} {self.ventas[i][0]:<10} {self.ventas[i][1]:<10} {self.ventas[i][2]:<10}")


ventas = Ventas()


print("Ventas iniciales:")
ventas.imprimir_ventas()


print("Buscar ventas:")
mes_buscar = int(input("Ingrese el mes (0-11): "))
dep_buscar = int(input("Ingrese el departamento (0: Ropa, 1: Deportes, 2: Juguetería): "))
resultado = ventas.buscar_ventas(mes_buscar, dep_buscar)
if resultado is not None:
    print("Ventas:", resultado)

print("Eliminar ventas:")
mes_eliminar = int(input("Ingrese el mes (0-11): "))
dep_eliminar = int(input("Ingrese el departamento a eliminar (0: Ropa, 1: Deportes, 2: Juguetería): "))
ventas.eliminar_ventas(mes_eliminar,dep_eliminar)
print("Ventas después de eliminar:")
ventas.imprimir_ventas()

File: arreglo.java
import java.util.Scanner;

public class arreglo {
    private int[][] ventas;
    private String[] meses;
    public arreglo() {
       
        ventas = new int[][]{
            {500, 300, 150},  
            {400, 250, 200},  
            {350, 450, 300},  
            {600, 350, 250},  
            {700, 300, 350},  
            {550, 400, 450},  
            {600, 500, 400},  
            {650, 550, 300},  
            {620, 350, 250},  
            {590, 300, 300},  
            {560, 270, 200},  
            {700, 350, 150}   
        };
        meses = new String[]{
            "Enero", "Febrero", "Marzo", "Abril", "Mayo", "Junio",
            "Julio", "Agosto", "Septiembre", "Octubre", "Noviembre", "Diciembre"
        };
    }

    public Integer buscarVentas(int mes, int departamento) {
        if (mes >= 0 && mes < 12 && departamento >= 0 && departamento < 3) {
            return ventas[mes][departamento];
        } else {
            System.out.println("Mes o departamento fuera de rango.");
            return null;
        }
    }

    public void eliminarVentas(int mes, int departamento) {
        if (mes >= 0 && mes < 12 && departamento >= 0 && departamento < 3) {
            ventas[mes][departamento] = 0;
        } else {
            System.out.println("Mes o departamento fuera de rango.");
        }
    }

    public void imprimirVentas() {
        System.out.printf("%-12s %-10s %-10s %-10s%n", "Mes", "Ropa", "Deportes", "Juguetería");
        for (int i = 0; i < 12; i++) {
            System.out.printf("%-12s %-10d %-10d %-10d%n",
                    meses[i], ventas[i][0], ventas[i][1], ventas[i][2]);
        }
    }

    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        arreglo ventas = new arreglo();

        System.out.println("Ventas iniciales:");
        ventas.imprimirVentas();

      
        System.out.println("Buscar ventas:");
        System.out.print("Ingrese el mes (0-11): ");
        int mesBuscar = scanner.nextInt();
        System.out.print("Ingrese el departamento (0: Ropa, 1: Deportes, 2: Juguetería): ");
        int depBuscar = scanner.nextInt();
        Integer resultado = ventas.buscarVentas(mesBuscar, depBuscar);
        if (resultado != null) {
            System.out.println("Ventas: " + resultado);
        }

      
        System.out.println("Eliminar ventas:");
        System.out.print("Ingrese el mes (0-11): ");
        int mesEliminar = scanner.nextInt();
        System.out.print("Ingrese el departamento a eliminar (0: Ropa, 1: Deportes, 2: Juguetería): ");
        int depEliminar = scanner.nextInt();
        ventas.eliminarVentas(mesEliminar, depEliminar);
        System.out.println("Ventas después de eliminar:");
        ventas.imprimirVentas();

        scanner.close();
    }
}
